### ISSUE TITLE
#enhancement : Change rst signal according to _neg / _pos

### ISSUE THREAD

**Issue by stdefeber** [esynr3z/corsair#63]
Currently the reset signal is always defined as "rst".
When one sets the register_reset  to "_neg" the signal does not reflect the active state anymore.

So would it be possible to make it "rstn" when register_reg = <>_neg ?

**Comment by esynr3z** [comment]
Yes it would be possible. For now, this was done intentionally in order to simplify templates. However, it looks like simple jinja macro is required.
